docs/governance/templates/release_issue.md

```diff
@@ -34,7 +34,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
   - [ ] Create a new file named {version}.md in `/site/content/en/blog/releases`. Copy the draft release content in this file (this will be what you send via email)[refer the previous release file].
   - run `make site-server` frequently to make sure everything looks fine for the release in your localhost
   - [ ] In `site/content/en/docs/Installation/_index.md #agones-and-kubernetes-supported-versions`, for the current version, replace `{{% k8s-version %}}` with hardcoded Kubernetes versions supported by the current version. And add a row for the Agones release version with `{{% k8s-version %}}` as its supported Kubernetes versions.
-  - [ ] Review all `link_test` and `data-proofer-ignore` attributes and remove for link testing [delete `data-proofer-ignore` from the release file of previous version]
+  - [ ] Run `make del-data-proofer-ignore FILENAME={version}-1.md` to remove `data-proofer-ignore` attribute from previous release blog. Review all occurrences of the link_test and data-proofer-ignore attributes globally. Exclude html and release files.
```

Comment:
On `FILENAME={version}-1.md` - why the `-1` ?

Reply:
In the previous release blog, there will be an occurrence of the 'data-proofer-ignore' attribute that requires removal. If the release version is 1.33.0 then we need to remove this attribute from 1.32.0.md.

Reply:
Oooooh! Of course, I was readying that as "append `-1` to the filename" not "subtract a minor version from `{version}`".

Thanks!